commands-yml/commands/interactions/touch/multi-touch-perform.yml

```diff
@@ -0,0 +1,114 @@
+---
+name: Touch Perform
+short_description: Perform a multi touch action sequence
+   
+
+example_usage:
+  java:
+    |
+      TouchAction actionOne = new TouchAction();
+      actionOne.press(10, 10);
+      actionOne.moveTo(10, 100);
+      actionOne.release();
+      TouchAction actionTwo = new TouchAction();
+      actionTwo.press(20, 20);
+      actionTwo.moveTo(20, 200);
+      actionTwo.release();
+      MultiTouchAction action = new MultiTouchAction();
+      action.add(actionOne);
+      action.add(actionTwo);
+      action.perform();
+  python:
+    |
+      actions = MultiTouchAction(driver)
```

Comment:
This is now how Python works for this. See https://github.com/appium/python-client/blob/master/test/functional/android/multi_action_tests.py

Reply:
Also would be good to have the import
```python
from appium.webdriver.common.touch_action import TouchAction
from appium.webdriver.common.multi_action import MultiAction
# ...
a1 = TouchAction()
# ...
a2 = TouchAction()
# ...
ma = MultiAction(self.driver)
ma.add(a1, a2)
ma.perform()
```